
{'date': 'Sat Jul 14 2018 17:31:03.587 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  m6n=http://images.tcdn.com.br/img/img_prod/419216/balao_de_decoracao_de_festa_aniversario_marshall_patrulha_canina_1872_3_20180312095941.jpg
           ^
SyntaxError: invalid syntax
'''},
{'date': 'Sat Jul 14 2018 17:31:36.138 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 2
  http://images.tcdn.com.br/img/img_prod/419216/balao_de_decoracao_de_festa_aniversario_marshall_patrulha_canina_1872_3_20180312095941.jpg
       ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 11:19:43.621 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 27
  noite = Cena(img = NOITE)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 22 2018 11:20:32.436 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    class Tabuleiro:
  module <module> line 30
    self.fase1= Cena(img=FASE1, tit = "SIMETRICO 3 X 4", style = dict(left=200, top="100px", width=540, height="320px"))
  module _spy.vitollino.main line 696
    Cena.c(**kwargs)
  module _spy.vitollino.main line 739
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
KeyError: img
'''},
{'date': 'Sun Jul 22 2018 11:21:49.516 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 30
  self.fase1= Cena(img=FASE1, tit = "SIMETRICO 3 X 4", style = dict(left=200, top="100px", width=540, height="320px"))
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 22 2018 11:23:54.906 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 47
    Tabuleiro ()
  module <module> line 28
    self.fase1= Cena(img=FASE1, tit = "SIMETRICO 3 X 4", style = dict(left=200, top="100px", width=540, height="320px"))
  module _spy.vitollino.main line 696
    Cena.c(**kwargs)
  module _spy.vitollino.main line 739
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
KeyError: img
'''},
{'date': 'Sun Jul 22 2018 11:36:13.49 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.noite = Cena(img = NOITE)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 22 2018 11:36:39.985 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.noite = Cena(img = NOITE)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 22 2018 11:37:20.336 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 35
  """ self.fase2= Cena(img=FASE2, tit = "SIMETRICO 3 X 4", style = dict(left=200, top="100px", width=540, height="320px"))
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Jul 22 2018 11:45:48.752 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  self..fase1.qverde = Elemento(img=QVERDE,style = dict(left=200, top="100px", width=20, height="20px"))
        ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 11:48:26.692 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 61
    Tabuleiro()
  module <module> line 32
    self.fase1.qverde.entra (self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 12:20:27.488 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 25
  def cenario(cenario):
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 22 2018 12:20:57.123 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  noite = Cena(img = NOITE)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Sun Jul 22 2018 12:24:20.650 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    Tabuleiro ()
  module <module> line 26
    noite.vai()
NameError: name 'noite' is not defined
'''},
{'date': 'Sun Jul 22 2018 12:24:58.849 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    Tabuleiro ()
  module <module> line 28
    fase1.entra (noite)
NameError: name 'fase1' is not defined
'''},
{'date': 'Sun Jul 22 2018 12:25:26.742 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    Tabuleiro ()
  module <module> line 28
    self.fase1.entra (noite)
NameError: name 'noite' is not defined
'''},
{'date': 'Sun Jul 22 2018 12:33:28.281 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 26
    noite.vai()
NameError: name 'noite' is not defined
'''},
{'date': 'Sun Jul 22 2018 12:36:24.657 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 12:37:14.717 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 12:45:06.78 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 12:49:13.509 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 12:55:12.537 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 13:09:43.239 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 13:14:00.614 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 14:21:45.734 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 14:22:33.707 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 14:23:38.620 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Tabuleiro ()
  module <module> line 30
    self.qsimbolo.entra(self.fase1)
  module _spy.vitollino.main line 453
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Elemento' object has no attribute 'nome'
'''},
{'date': 'Sun Jul 22 2018 14:55:59.261 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  self.qsimbolo = Elemento(img = QSIMBOLO, style = dict(for simbolo in range (4):
                                                                                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 14:56:27.525 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  self.qsimbolo = Elemento(img = QSIMBOLO,style = dict(for simbolo in range (4):
                                                                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 14:57:50.478 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  self.qsimbolo = Elemento(img = QSIMBOLO,style = dict(
                                                                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 15:01:03.38 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.qsimbolo = Elemento(img = QSIMBOLO,style = dict(
                                                                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 15:13:33.832 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.qsimbolo = Elemento(img = QSIMBOLO,style = dict(
                                                                                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 15:36:32.459 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 37
  """### TABULEIRO RESPOSTA ####
  ^
SyntaxError: invalid syntax : triple string end not found
'''},
{'date': 'Sun Jul 22 2018 15:36:56.851 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 37
  """### TABULEIRO RESPOSTA ####
  ^
SyntaxError: invalid syntax : triple string end not found
'''},
{'date': 'Sun Jul 22 2018 15:39:03.180 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 37
  """### TABULEIRO RESPOSTA ####
  ^
SyntaxError: invalid syntax : triple string end not found
'''},
{'date': 'Sun Jul 22 2018 15:39:30.792 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 38
  TBRX, TBRY = 200, 100
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Jul 22 2018 15:59:16.830 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.cobre1 = Elemento(QSIMBOLO,style = dict( left=100, top="100px", width=540, height="320px"))self.cobre1.
                                                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 15:59:50.1 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 40
  self.fase2.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Jul 22 2018 16:00:46.699 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 85
    Tabuleiro ()
  module <module> line 30
    self.cobre.entra(self.fase1)
AttributeError: 'Tabuleiro' object has no attribute 'cobre'
'''},
{'date': 'Sun Jul 22 2018 16:03:31.25 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.cobre1 = Elemento(QBRANCO,style = dict( for cobre in range(5):
                                                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 16:04:36.281 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.cobre1 = Elemento(QBRANCO, style = dict( for cobre in range(5):
                                                                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 16:05:30.366 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 29
  self.cobre1 = Elemento(QBRANCO,for cobre in range(5):
                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 22 2018 16:17:41.198 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 85
    Tabuleiro ()
  module <module> line 29
    self.cobre1 = Elemento(QBRANCO, style = dict(left= cobre + 100, top= "100px", width=40, height="40px"))
NameError: name 'cobre' is not defined
'''},
{'date': 'Sun Jul 22 2018 16:24:58.446 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 93
    Tabuleiro ()
  module <module> line 33
    	self.cobre1 = Elemento(QBRANCO, style = dict(
NameError: name 'cobre1' is not defined
'''},
{'date': 'Sun Jul 22 2018 16:30:20.493 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.cobre1 = Elemento(QBRANCO, style = dict(
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Jul 22 2018 16:30:54.752 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.linha = Elemento(QBRANCO, style = dict(
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Jul 22 2018 16:31:22.470 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.linha = Elemento(QBRANCO, style = dict(
  ^
IndentationError: unexpected indent
'''},
{'date': 'Sun Jul 22 2018 16:32:04.445 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.linha = Elemento(QBRANCO, style = dict(
  ^
IndentationError: unexpected indent
'''},